tests/test_smoke.py

```diff
@@ -2167,6 +2236,19 @@ def test_nonexistent_bucket(self, nonexist_bucket_url):
                 endpoint_url = cloudflare.create_endpoint()
                 command = f'AWS_SHARED_CREDENTIALS_FILE={cloudflare.R2_CREDENTIALS_PATH} aws s3api head-bucket --bucket {nonexist_bucket_name} --endpoint {endpoint_url} --profile=r2'
                 expected_output = '404'
+            elif nonexist_bucket_url.startswith('cos'):
+                # Using API calls, since using rclone requires a profile's name
```

Comment:
Would you mind elaborating on this? Can we not fetch rclone's IBM COS profile to run this test?